### ISSUE TITLE
Change x-axis scale from steps to epochs

### ISSUE THREAD

**Issue by schrum2** [schrum2/MarioDiffusion#48]
Epochs is more understandable. Steps is confusing since it depends on the training set size. It's unclear how close a run is to finishing.

This would be a change to the loss_plotter.py file, which should probably also have its name changed to simply plotter.py (be sure to update it everywhere)
